Docs/userdeveloperguide.tex

```diff
@@ -244,5 +245,10 @@ \section*{Test Cases:}
 
 This folder contains two samples of data for use in the PACCMAN program.  The data contained by the "xucomparison" file contains values from the 2001 paper: The Design of Conformal Cooling Channels in Injection Molding Tooling" by Xiorong Xu, Emanuel Sachs, and Samuel Allen. "xucomparison.csv" is a table produced through WebPlotDigitizer of Figure 4 in the Xu paper. The data contained by the "guilongcomparison" file is from the 2010 paper "Analysis of thermal cycling efficiency and optimal design of heating/coolingsystems for rapid heat cycle injection molding process" by Wang Guilong, Zhao Guoqun, Li Huiping, and Guan Yanjin. "guilongcomparison.csv" contains the data from Figure 6 pulled with WebPlotDigitizer. There are also .png files graphically comparing the real world results of these papers to the program's created graphs.
```

Comment:
Please split the sentences into individual lines (i.e. just have it so they are 245, 246, 247, etc) and use the \cite command to inline reference the papers.